test/posthog_test.exs

```diff
@@ -121,13 +119,11 @@ defmodule PostHogTest do
       assert %{
                event: "case tested",
                distinct_id: "distinct_id",
-               properties: %{struct: %{hello: nil}, ref: _},
+               properties: %{struct: %{hello: nil}, ref: _} = properties,
```

Comment:
I removed the assignment here because `properties` is not being used and I saw warnings when running the test cases locally. We should remove it if there's no use for it

Reply:
Yeah the usage was lost during `JSON` -> `LoggerJSON.RedactorEncoder`, but it is used again now!